test/unit/pharmacies/transform.js

```diff
@@ -147,12 +148,13 @@ describe('Profiles transform', () => {
     const otAltOffset = transformedData[0].openingTimesAlterationsAsOffset;
     // eslint-disable-next-line
     expect(otAltOffset).to.exist;
-    expect(otAltOffset.length).to.equal(5);
+    expect(otAltOffset.length).to.equal(6);
     expectAltOpenClose(otAltOffset[0], '2014-05-05', 660, 960);
     expectAltOpenClose(otAltOffset[1], '2014-05-26', 660, 960);
     expectAltOpenClose(otAltOffset[2], '2014-04-18', 660, 960);
     expectAltOpenClose(otAltOffset[3], '2014-04-21', 660, 960);
     expectAltOpenClose(otAltOffset[4], '2014-08-25', 660, 960);
+    expectAltOpenClose(otAltOffset[5], '2015-01-01', undefined, undefined);
```

Comment:
Can a specific test be added for this scenario please? In order to make it clear this is what represents the data for a closed alteration? I think by doing that the comment in `transform.js` could be removed.